docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -106,6 +106,13 @@ As a result the delay can range from 3 to 10 days or even more. We check for
 updates every day and provide the most up-to-date data.
 
 ## Limitations
+
+Between May 13 2024 and August 6 2024, signal values were 25%-50% lower compared to previous time periods.
+This affected _all_ signals and symptom sets.
```

Comment:
i mightve added this for clarity
```suggestion
This affected _all_ `google-symptoms` signals and symptom sets.
```

Reply:
Addressed in https://github.com/cmu-delphi/delphi-epidata/pull/1591